libs/login/WordPressLoginFlow/build.gradle

```diff
@@ -21,10 +21,10 @@ apply plugin: 'kotlin-kapt'
 apply plugin: 'kotlin-android-extensions'
 
 repositories {
+    mavenCentral()
```

Comment:
The build file changes have been made as part of several WCAndroid PRs. Most of them are version upgrades, so as long as we test the login flow, should be good to go. The `mavenCentral` addition and the `bintray` maven url removal are related to Bintray shutdown.